docs/site/Model.md

```diff
@@ -407,12 +389,42 @@ class MyUserModel extends Entity {
 }
 ```
 
-### Property Decorator
+### Scope
+
+_Scope_ enable you to specify commonly-used queries that you can reference as method calls on a model.
+
+If you wish for a scope to be applied across all queries to the model, you can set, for example:
+
+```ts
+{
+  name: "Product",
+  scope: {
```

Comment:
```suggestion
  scopes: {
```

Reply:
And I think you need to give each scope a name?
Like 

```js
"scopes": {
	"vips": {"where": {"vip": true}}, 
	"top5": {"limit": 5, "order": "age"}
}

```

Reply:
I don't think this part works in LB4, at least from what I tried.

Reply:
👍 take my comment back :p

Reply:
Just want to double check this part: in LB3 we allow:
```ts
"scopes": {
	"vips": {"where": {"vip": true}}, 
	"top5": {"limit": 5, "order": "age"}
}
```
then the scopes can be used as 
```ts
User.scope("vips", { where: { vip: true } });
User.scope("top5", { limit: 5, order: "age" });
```
I don't think this is valid in LB4 because:
- `User` model, or any lb4 model, doesn't have such method `scope`
- LB4 uses repositories to query data. So `User.scope` won't work.

Is this correct?

Reply:
Defining 
```ts
"scopes": {
	"vips": {"where": {"vip": true}}, 
	"top5": {"limit": 5, "order": "age"}
}
```
is equivalent to 
```ts
// please note User.scope defines scope, not apply scope
User.scope("vips", { where: { vip: true } });
User.scope("top5", { limit: 5, order: "age" });
```
After defining them, you can call APIs like `User.vips()` to query with the predefined filter.

But your conclusion is correct 👌 we don't have such APIs in User repository. Therefore `scopes` won't be supported. Only specifying the default `scope` works.